tests/test_spaces.py

```diff
@@ -1,34 +1,38 @@
+import gym
```

Comment:
This file makes no sense. We don't need to test gym.Spaces, it's assumed to work. rllab.Spaces still exists and should be tested.

We might want to test rllab.envs.util, but that's a different test.

Reply:
restored.